dpnp/dpnp_iface_arraycreation.py

```diff
@@ -710,6 +711,38 @@ def geomspace(start, stop, num=50, endpoint=True, dtype=None, axis=0):
     return call_origin(numpy.geomspace, start, stop, num, endpoint, dtype, axis)
 
 
+def identity(n, dtype=None, *, like=None):
+    """
+    Return the identity array.
+
+    The identity array is a square array with ones on the main diagonal.
+
+    For full documentation refer to :obj:`numpy.identity`.
+
+    Limitations
+    -----------
+    Parameter ``like`` is currently not supported .
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> np.identity(3)
+    array([[1.,  0.,  0.],
+           [0.,  1.,  0.],
+           [0.,  0.,  1.]])
+
+    """
+    if not use_origin_backend():
+        if like is not None:
+            pass
```

Comment:
```suggestion
        if n < 0:
            pass
        elif like is not None:
            pass
```
`n` should be non-negative.

Reply:
outdated.